mock-rollout/src/Products/ProductContext.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { createContext, FC, useEffect, useState } from 'react';
+import { IProduct } from './ProductModel';
+import { ProductService } from './ProductService';
+
+interface Props  {
```

Comment:
what props? usually interface naming and defitionon looks like this:

export interface IProductProps (I - because it's an interface, Product - the refering entity, and Props - what it actually is). You'll have a lot of Props if you only call them like this.